dnn/data/augmentation/detection.py

```diff
@@ -221,6 +233,10 @@ def __call__(self, data, training=True):
         # apply transformation
         img_hw3 = tfa.image.transform(img_hw3, affine_coeffs, 
             interpolation='BILINEAR', output_shape=tf.cast(self.output_hw, tf.int32))
+        img_hw3 = tf.image.random_saturation(img_hw3, self.sat[0], self.sat[1])
+        img_hw3 = tf.image.random_hue(img_hw3, self.hue)
+        img_hw3 = tf.image.random_contrast(img_hw3, self.contrast[0], self.contrast[1])
+        img_hw3 = tf.image.random_brightness(img_hw3, self.brightness)
```

Comment:
I think image-level augmentations should be moved to its own module, in parallel to `DetectionAugmentor`. May be create another `tf.Module` subclass in somewhere like `dnn/data/augmentation/image.py`?

Reply:
Done, moved into image.py